packages/react-server-cli/test/parseCliArgs.js

```diff
@@ -0,0 +1,326 @@
+import test from 'ava';
```

Comment:
Oh, man this is so great!  Thanks @jbenesch for working this out and locking it down with these tests!